react/features/base/react/components/native/headerstyles.js

```diff
@@ -0,0 +1,81 @@
+// @flex
+
+import { StyleSheet } from 'react-native';
+
+import { ColorSchemeRegistry, schemeColor } from '../../../color-scheme';
+import { BoxModel } from '../../../styles';
+
+const HEADER_FONT_SIZE = 18;
+const HEADER_HEIGHT = 48;
```

Comment:
this used to be 56, why the change?

Reply:
That's main the point of the commit "Make the header more compact".

The header was unnecessarily tall and that was very visible when someone wanted to apply a more native color scheme (for example iOS colors)

Reply:
MD defines it as 56: https://material.io/design/components/app-bars-top.html#specs I know we are not implementing MD we are "getting inspired" by it. So are we ok with this change?

Reply:
Well, I really love this new compact header, for me it looks good on Android, but it definitely looks way better on iOS. I'd say it's less platform specific now